### ISSUE TITLE
支持llama3么，有个小程序接入模型接口，不包括llama，想用openai解决，第三方买的llama APPI

### ISSUE THREAD

**Issue by onlyhuman028** [songquanpeng/one-api#1419]
**例行检查**

[//]: # (方框内删除已有的空格，填 x 号)
+ [ ] 我已确认目前没有类似 issue
+ [ ] 我已确认我已升级到最新版本
+ [ ] 我已完整查看过项目 README，已确定现有版本无法满足需求
+ [ ] 我理解并愿意跟进此 issue，协助测试和提供反馈
+ [ ] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**功能描述**

**应用场景**

**Comment by QAbot-zh** [comment]
llama3 支持的

**Comment by onlyhuman028** [comment]
收到，已经完成部署，openai和grok 测试都通过。但是有一个 deepinfra.com的账号，购买了key 。在oneapi 的接口怎么填都通不过测试。我填的是 
  https://api.deepinfra.com/v1。官方指导文件：https://deepinfra.com/meta-llama/Meta-Llama-3-70B-Instruct/api?example=js-text-gen。 谢谢先

**Comment by onlyhuman028** [comment]
是不是模型名称问题

**Comment by QAbot-zh** [comment]
看了下例程，是调用格式跟OpenAI不匹配，得让oneapi适配这个渠道了

**Comment by onlyhuman028** [comment]
> 看了下例程，是调用格式跟OpenAI不匹配，得让oneapi适配这个渠道了

哦，谢谢。您的意思是 需要oneapi 官方适配一下grok ，oneapi的用户才能正常用？

**Comment by onlyhuman028** [comment]
up

**Comment by tanlentboy** [comment]
确实, 我也遇到这个问题了. deepinfra.com 上有最新的 nvidia/Nemotron-4-340B-Instruct  模型.    但是在oneapi上确实无法使用.

**Comment by kaikanertan** [comment]
> > 看了下例程，是调用格式跟OpenAI不匹配，得让oneapi适配这个渠道了
> 
> 哦，谢谢。您的意思是 需要oneapi 官方适配一下deepinfra.com ，oneapi的用户才能正常用？

在one-api中可以尝试自定义渠道，baseurl填 https://api.deepinfra.com/v1/openai
deepinfra的doc里写它们兼容openai的baseurl是 https://api.deepinfra.com/v1/openai

**Comment by kaikanertan** [comment]
> 确实, 我也遇到这个问题了. deepinfra.com 上有最新的 nvidia/Nemotron-4-340B-Instruct 模型. 但是在oneapi上确实无法使用.

在one-api中可以尝试自定义渠道，baseurl填 https://api.deepinfra.com/v1/openai
deepinfra的doc里写它们兼容openai的baseurl是 https://api.deepinfra.com/v1/openai

**Comment by kaikanertan** [comment]
> 确实, 我也遇到这个问题了. deepinfra.com 上有最新的 nvidia/Nemotron-4-340B-Instruct 模型. 但是在oneapi上确实无法使用.

还有一家nebius.
https://docs.nebius.com/studio/inference/models

**Comment by QAbot-zh** [comment]
一种补救方法：

![image](https://github.com/user-attachments/assets/e31d9bc2-789b-4a35-a1aa-8a61e850435b)

**Comment by kaikanertan** [comment]
> 一种补救方法：
> 
> ![image](https://private-user-images.githubusercontent.com/40236765/396757632-e31d9bc2-789b-4a35-a1aa-8a61e850435b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ5Mjc3MDUsIm5iZiI6MTczNDkyNzQwNSwicGF0aCI6Ii80MDIzNjc2NS8zOTY3NTc2MzItZTMxZDliYzItNzg5Yi00YTM1LWExYWEtOGE2MWU4NTA0MzViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIzVDA0MTY0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0MjNmNGVhMjg2NDhiY2MzYzIzMzU4ZjRkZjg1MWE5OGRkNmM2NTRiYWUxYzc5MTUzN2UyNjZjODQwNjExNmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5jU6SY1K6wCqT1nUy0Uclow4GUz-zdkmArrEtI3rA_U)

baserul的最后为什么需要一个 # ? 笔误 or 确实需要？

**Comment by QAbot-zh** [comment]
> baserul的最后为什么需要一个 # ? 笔误 or 确实需要？

# 用来屏蔽掉默认的补全（/v1/chat/completions）

**Comment by kaikanertan** [comment]
> > baserul的最后为什么需要一个 # ? 笔误 or 确实需要？
> 
> “#” 是用来屏蔽掉默认的补全（/v1/chat/completions）

😃 🫰🏻
